README.md

```diff
@@ -1,53 +1,40 @@
--------------
-GeoToad is an open-source query tool to query the geocaching.com website.
-It supports exporting your results into a dozen or more formats such as
-GPX (also variants suited for GSAK, or your road nav device) and HTML.
-
-
--------------------
-System Requirements
--------------------
- - Windows
-   - Windows 98 or Higher with
-     - a sufficiently recent Ruby version
-     - a SSL certificate chain required for HTTPS (also provided with GeoToad)
- - Mac OS
-   - Mac OS X 10.4 or Higher
- - Other Operating Systems
-   - You may need to install Ruby before running GeoToad.
-     - RedHat/Fedora Linux: built-in
-     - Debian/Ubuntu Linux: apt-get install ruby libopenssl-ruby, then dpkg -i geotoad_*.deb
-     - FreeBSD: pkg_add -r ruby
-
-Optionally, if you have gpsbabel (=> http://www.gpsbabel.org/)
-GeoToad can output to over a dozen additional GPS formats.
-
-
----------------
-Getting Started
----------------
-The easiest way to get started with GeoToad is to use the "text user
-interface" (TUI). Simply double click on the GeoToad program icon, or from
-a command prompt run "geotoad". On UNIX, you may have to run "./geotoad.rb".
+# GeoToad
+by Thomas Stromberg and The GeoToad Project © 2012–2016
+
+## Table of Contents:
+* [About GeoToad](#about-geotoad)
+* [System Requirements](#system-requirement)
+* [Getting Started](#getting-started)
+* [Command-Line Syntax](#command-line-syntax)
+* [Command-Line Examples](#command-line-exampes)
+
+## About GeoToad
+GeoToad is an open-source query tool to query the geocaching.com website. It supports exporting your results into a dozen or more formats such as GPX (also variants suited for GSAK, or your road nav device) and HTML.
+
+## System Requirements
+#### Windows:
+* Windows 98 or Higher with
+  * a sufficiently recent Ruby version
+  * a SSL certificate chain required for HTTPS (also provided with GeoToad)
+
+#### macOS
+   * macOS 10.4 or Higher. If you have [Homebrew](https://brew.sh), install with `brew install https://github.com/steve8x8/geotoad/macOS/geotoad.rb`.
+
+#### Other Operating Systems
+* You may need to install Ruby before running GeoToad.
+  * **RedHat/Fedora Linux**: built-in
+  * **Debian/Ubuntu Linux**: `apt-get install ruby libopenssl-ruby && dpkg -i geotoad_*.deb`
+  * **FreeBSD**: `pkg_add -r ruby`
+
+Optionally, if you have [gpsbabel](http://www.gpsbabel.org/), GeoToad can output to over a dozen additional GPS formats.
+
+## Getting Started
+The easiest way to get started with GeoToad is to use the "text user interface" (TUI). Simply double click on the GeoToad program icon, or from a command prompt run `geotoad`. On UNIX, you may have to run `./geotoad.rb`.
 You will then see a screen that looks similar to this:
 
+```
 ==============================================================================
-:::                // GeoToad %VERSION% Text User Interface //                :::
+:::                // GeoToad %VERSION% Text User Interface //             :::
```

Comment:
This is how it was in the previous text, so the packaging tool did not change it there. It seemed like you wanted this here since that information is not important in the README. What do you want it changed to?